modules/virt-create-node-network-config-console.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: PROCEDURE
+[id="virt-create-node-network-config-console_{context}"]
+= Creating an interface on nodes
+
+You can create a policy by using either a form or YAML in the web console.
+
+.Procedure
+. Navigate to *Networking* → *NodeNetworkConfigurationPolicy*.
+
+. In the *NodeNetworkConfigurationPolicy* page, click *Create*, and select *From Form* option.
++
+In case there are no existing policies, you can alternatively click *Create NodeNetworkConfigurationPolicy* to create policy using form.
++
+[NOTE]
+====
+To create policy using YAML, click *Create*, and select *With YAML* option.
+====
+
+. Optional: Check the *Apply this NodeNetworkConfigurationPolicy only to specific subsets of nodes using the node selector* checkbox to specify the nodes where the policy must be applied.
+
+. Enter the policy name in *Policy name* field.
+
+. Optional: Enter the description of the policy in *Description* field.
+
+. Optional: In the *Policy Interface(s)* section, by default, a bridge interface is added. Update the list of network interfaces by executing the following steps:
+
+.. Enter the name of the interface in *Interface name* field.
+
+.. Select the network stage from *Network state* dropdown. The default selected value is *Up*.
+
+.. Select the type of interface from *Type* dropdown. The available values are *Bridge*, *Bonding*, and *Ethernet*. The default selected value is *Bridge*.
++
+[NOTE]
+====
+To add a VLAN interface, you must use YAML to create the policy. Addition of a VLAN interface using the form is not supported.  Once you have added a VLAN interface, you cannot edit the policy using form.
+====
+
+.. Optional: In the IP configuration field, check *IPv4* checkbox to assign an IPv4 address to the interface, and configure the IP address assignment details:
```

Comment:
Again, it's not optional.

Reply:
As per my understanding and discussion with @upalatucci , the user can leave the checkbox unchecked if they choose not have an IP address assigned, which makes this step optional. Any different thoughts?

Reply:
Yes @snarayan-redhat IPv4 can be unchecked. This means that the interface will not have an ipv4 address